classroom-template-bundle-spec.md

```diff
@@ -0,0 +1,168 @@
+# Qwiklabs ClassroomTemplate Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
```

Comment:
Since this is a public doc, we should explicitly call out that this is pseudo-deprecated and integrators should use CourseTemplates if possible.

```suggestion
> **Note: For Interim Use Only**
>
> This specification is for existing Qwiklabs platform users that are currently using the `ClassroomTemplate` model. It should be used to help transition from `ClassroomTemplates` to [`CourseTemplates`](./course-template-bundle-spec.md). If you are not currently using `ClassroomTemplates`, please use `CourseTemplates` instead.

```